appveyor.yml

```diff
@@ -35,14 +35,10 @@ test_script:
   - node --version
   - yarn lint
   - yarn test
-#  - yarn test-e2e
-#  - yarn package-dev
-#  - 7z a development.zip release
-#  - rmdir /S /Q release
-  - cmd: IF /I "%ROLLBAR_POST_TOKEN%"=="" ( ECHO Missing Rollbar POST token; no package built.) ELSE ( yarn package )
-#  - 7z a production.zip release
-  # submit uploader release to antivirus vendors if tagged
-  - cmd: IF /I %APPVEYOR_REPO_TAG% == True ( yarn av-whitelist ) ELSE ( echo Not a tagged release. )
+  # submit uploader release to antivirus vendors and also publish to S3 if tagged
+  - cmd: IF /I "%ROLLBAR_POST_TOKEN%"=="" ( ECHO Missing Rollbar POST token; no package built. ) ELSE ( yarn package )
+  - cmd: IF /I %APPVEYOR_REPO_TAG% == True ( yarn publish-s3 ) ELSE ( echo Not a tagged release. )
+# TODO: put this back: & yarn av-whitelist
```

Comment:
Is this `TODO` for a later date?

Reply:
Oops, looks like I didn't push the last commit to add it back.